.rubocop.yml

```diff
@@ -1,54 +1,54 @@
-AllCops:
-  NewCops: enable
-  Exclude:
-    - "Guardfile"
-    - "Rakefile"
-    - "node_modules/**/*"
-
-  DisplayCopNames: true
-
-Layout/LineLength:
-  Max: 120
-Metrics/MethodLength:
-  Max: 20
-Metrics/AbcSize:
-  Max: 50
-Metrics/ClassLength:
-  Max: 150
-Metrics/BlockLength:
-  IgnoredMethods: ['describe']
-  Max: 30
-
-
-Style/Documentation:
-  Enabled: false
-Style/ClassAndModuleChildren:
-  Enabled: false
-Style/EachForSimpleLoop:
-  Enabled: false
-Style/AndOr:
-  Enabled: false
-Style/DefWithParentheses:
-  Enabled: false
-Style/FrozenStringLiteralComment:
-  EnforcedStyle: never
-
-Layout/EndOfLine:
-  EnforcedStyle: crlf
-Layout/HashAlignment:
-  EnforcedColonStyle: key
-Layout/ExtraSpacing:
-  AllowForAlignment: false
-Layout/MultilineMethodCallIndentation:
-  Enabled: true
-  EnforcedStyle: indented
-Lint/RaiseException:
-  Enabled: false
-Lint/StructNewOverride:
-  Enabled: false
-Style/HashEachMethods:
-  Enabled: false
-Style/HashTransformKeys:
-  Enabled: false
-Style/HashTransformValues:
-  Enabled: false
+AllCops:
+  NewCops: enable
+  Exclude:
+    - "Guardfile"
+    - "Rakefile"
+    - "node_modules/**/*"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 20
+Metrics/AbcSize:
+  Max: 50
+Metrics/ClassLength:
+  Max: 150
+Metrics/BlockLength:
+  IgnoredMethods: ['describe']
+  Max: 30
+
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Hi @HAMMAS-SALEEM , thanks a lot for your suggestion! We'll work on it. ^^